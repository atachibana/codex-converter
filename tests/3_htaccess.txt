Reset your permalinks through '''Settings > Permalinks.''' If this does not work, you may have to edit the <tt>.htaccess</tt> file manually. 

<pre id="htaccess_sample"># BEGIN WordPress
<IfModule mod_rewrite.c>
RewriteEngine On
RewriteBase /
RewriteRule ^index\.php$ - [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FLENAME} !-d
RewriteRule . /index.php [L]
</IfModule>
# END WordPress</pre> 

If you are not familiar with editing your <tt>.htaccess</tt> file, contact your hosting provider to ask them to turn on mod_rewrite rules.