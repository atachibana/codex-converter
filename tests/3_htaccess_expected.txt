<p>Reset your permalinks through <strong>Settings > Permalinks.</strong> If this does not work, you may have to edit the <code>.htaccess</code> file manually. </p>
<p></p>
[code language="php"]# BEGIN WordPress
&lt;IfModule mod_rewrite.c&gt;
RewriteEngine On
RewriteBase /
RewriteRule ^index\.php$ - [L]
RewriteCond %{REQUEST_FILENAME} !-f
RewriteCond %{REQUEST_FLENAME} !-d
RewriteRule . /index.php [L]
&lt;/IfModule&gt;
# END WordPress [/code]
<p></p>
<p>If you are not familiar with editing your <code>.htaccess</code> file, contact your hosting provider to ask them to turn on mod_rewrite rules.</p>